tests/test_calendar_privacy.py

```diff
@@ -0,0 +1,43 @@
+from app.internal.calendar_privacy import can_show_calendar
+# TODO after user system is merged:
```

Comment:
you can place it!

Reply:
As mentioned-  what I need has yet to be merged:
(current logged user)

https://github.com/PythonFreeCourse/calendar/pull/293/